docs/pages/addons/api/index.md

```diff
@@ -26,7 +26,7 @@ See how we can use this:
 
 ```js
 // Register the addon with a unique name.
-addonAPI.register('kadira/notes', (storybookAPI) => {
+addonAPI.register('my-organisation/my-addon', (storybookAPI) => {
```

Comment:
@ndelangen can we put this in a different PR? i think it is unrelated to eslint?